lib/SnippetsTestMessageProvider.jsm

```diff
@@ -219,6 +219,34 @@ const MESSAGES = () => ([
       "test": "takeover",
     },
   },
+  {
+    "id": "SIMPLE_TEST_WITH_SECTION_HEADING",
+    "template": "simple_snippet",
+    "content": {
+      "button_label": "Get one now!",
+      "button_url": "https://www.mozilla.org/en-US/firefox/accounts",
+      "icon": TEST_ICON,
+      "title": "Firefox Account!",
+      "text": "<syncLink>Sync it, link it, take it with you</syncLink>. All this and more with a Firefox Account.",
+      "links": {"syncLink": {"url": "https://www.mozilla.org/en-US/firefox/accounts"}},
+      "block_button_text": "Block",
+      "section_title_icon": TEST_ICON_16,
```

Comment:
Just curious, this intended to be used as a full-colour icon or one of the black-and-white icons like we have for other sections?

Reply:
I _think_ they want it to generally match the style of the other B&W icons like in the other sections.

Reply:
We talked about this too – in order to support dark theme it might be worth shipping some icons that would support dark/light theme or have some kind of css solution for that, but I kinda want to ask giorgos about that (usually we go with data URIs)

Reply:
Let's leave this as-is for now. Can you switch the value of `TEST_ICON_16` to `"resource://activity-stream/data/content/assets/glyph-pocket-16.svg"` to make it more realistic?